runPerftool.psm1

```diff
@@ -442,7 +442,10 @@ Function ProcessToolCommands{
                 $recvCmd =  $recvCmd -ireplace [regex]::Escape($CommandsDir), "$RecvDir/Receiver"
                 LogWrite "Invoking Cmd - Machine: $recvComputerName Command: $recvCmd" 
                 $recvJob = Invoke-Command -Session $recvPSSession -ScriptBlock ([Scriptblock]::Create($recvCmd)) -AsJob 
-    
+                
+                if ($Toolname -eq "ntttcp") {
+                    Start-Sleep -Seconds 5
```

Comment:
Should we use the $poll time (param) passed into the function here instead of hardcoding to 5?
Also I noticed a start-sleep 60 in this same function. I think we could change that to use $TimeoutValueBetweenCommandPairs   [which defaults to 60 today]

Reply:
Yes, switched to the pollTime variabel that is 5 seconds.